img/c/caffeine.tv.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 37 28.811" width="31.158" height="27.116"><path d="m27.927 19.85 7.282-4.207-7.282 12.617zm-6.533 3.772 5.087 5.876H11.218Zm-12.32-7.115V28.26L1.441 15.038l7.631 1.47zm0-7.546L1.79 13.167 9.073.55v8.411Zm6.532-3.773L10.52-.688h15.263zm12.32 7.115V.55l7.632 13.222-7.631-1.469zm0 6.5v-5.577l7.242 1.394-7.241 4.182zm-.906 1.57v8.363l-4.827-5.576zM16.937 18.02l9.257 1.783-4.924 2.843-.002.001-.003.002-10.5 6.064zM9.98 16.132V10.44l6.172 7.128L9.98 28.26zm-.907-6.124v5.576l-7.24-1.394zm.907-1.57V.074l4.827 5.576zm10.083 2.352-9.257-1.783 4.93-2.846.003-.002L26.235.097zm6.957 1.888v5.693l-6.171-7.128L27.02.55zm-10.075 4.42-6.22-7.183 9.33 1.796 6.22 7.184-9.33-1.796zm19.952-2.647.002-.01v-.014a.45.45 0 0 0 0-.02l.001-.01v-.015a.44.44 0 0 0-.002-.017l-.001-.013-.002-.014a.687.687 0 0 0-.003-.016l-.002-.014-.004-.013a.512.512 0 0 0-.004-.015l-.004-.015-.004-.012a.547.547 0 0 0-.006-.016l-.006-.014a.256.256 0 0 0-.013-.026.35.35 0 0 0-.007-.014l-.003-.005-8.973-15.546c0-.002-.002-.003-.003-.004a.569.569 0 0 0-.018-.03l-.007-.008-.01-.014-.01-.012-.008-.01-.011-.01a.46.46 0 0 0-.01-.01l-.012-.01-.01-.01a.396.396 0 0 0-.012-.01l-.013-.01-.009-.007a.473.473 0 0 0-.014-.01l-.015-.009-.004-.002-.004-.002-.016-.009-.015-.007-.01-.005-.016-.006a.384.384 0 0 0-.014-.006l-.012-.004-.015-.004a.639.639 0 0 0-.014-.004l-.015-.004-.013-.002a.494.494 0 0 0-.015-.003l-.017-.002h-.011a.415.415 0 0 0-.035-.003H9.506l-.019.002h-.01l-.016.003a.517.517 0 0 0-.017.003l-.012.002-.015.004a.416.416 0 0 0-.015.004l-.013.003-.014.005a.447.447 0 0 0-.014.006l-.013.005-.013.006a.427.427 0 0 0-.015.007l-.012.006-.01.007a.556.556 0 0 0-.017.01l-.011.007-.009.006a.484.484 0 0 0-.016.013l-.011.008-.004.004-.004.003a.459.459 0 0 0-.025.024l-.008.01a.37.37 0 0 0-.046.06.45.45 0 0 0-.01.014l-.003.005L.161 14.178l-.003.005a.539.539 0 0 0-.015.03l-.006.012a.431.431 0 0 0-.028.09v.004l-.002.005-.002.014a.426.426 0 0 0-.002.02l-.002.011v.033l-.001.012v.013l.002.018.001.013.002.014.003.016.002.014.004.013.004.015.004.015.004.011.006.017.006.014.005.01.008.016.007.014.003.005 8.973 15.546.002.004a.43.43 0 0 0 .021.032l.01.014.005.006a.44.44 0 0 0 .03.033l.005.006.008.008.018.015.014.012.01.008a.448.448 0 0 0 .01.007l.023.015.01.006.005.004a.402.402 0 0 0 .047.022l.003.001.005.002c.014.006.028.01.043.015l.016.004a.492.492 0 0 0 .036.007l.018.003.044.003h17.959l.04-.002h.01l.012-.003a.435.435 0 0 0 .021-.003l.017-.004.012-.003.012-.003.018-.006a.452.452 0 0 0 .022-.008l.006-.002a.436.436 0 0 0 .033-.016l.015-.007.005-.003.017-.01.016-.01.01-.008.01-.009.014-.01.004-.004.01-.01.003-.002.01-.009.006-.006.014-.016.015-.017.004-.005.01-.014a.494.494 0 0 0 .022-.033l.002-.004 8.969-15.538.004-.008c.002-.002.002-.005.004-.006a.45.45 0 0 0 .02-.044c.003-.006.006-.01.008-.017l.003-.009a.492.492 0 0 0 .006-.017l.004-.013.002-.01a.365.365 0 0 0 .005-.02l.001-.005v-.006l.003-.013a.432.432 0 0 0 .002-.02z" style="stroke:#000;stroke-width:.2"/></svg>
```

Comment:
I've emailed them to ask for an SVG with edges that doesn't look like mount Everest.
![screenshot_1644406598](https://user-images.githubusercontent.com/3535780/153194120-a54d6f7e-7e7d-4553-8fc0-a044593d73c0.png)